python/dolfin/function/expression.py

```diff
@@ -7,6 +7,7 @@
 from typing import Callable
 
 import numba
+import numba.ccallback
```

Comment:
Is this needed? Fully qualified namespace is used below.

Reply:
Seems needed, numba.ccallback doesn't get imported with numba.